lib/rspec/rails/adapters.rb

```diff
@@ -201,7 +203,13 @@ def assertion_method_names
             select do |m|
               m.to_s =~ /^(assert|flunk|refute)/
             end
-          methods + [:build_message]
+          # Starting on Rails 4, Minitest is the default testing framework so no
+          # need to add TestUnit specific methods.
+          if ::Rails::VERSION::STRING >= '4.0.0'
```

Comment:
As a point of style, it's nice to write method behavior that depends on constants by putting the conditional statement outside of the method definition so that it doesn't need to get calculated every time the method is called.

```ruby
if ::Rails::VERSION::STRING >= '4.0.0'
  def assertion_method_names
    […]
  end
else
  def assertion_method_names
    […]
  end
end
```

In this case, `define_assertion_delegators` only gets called when the concern is included, which is just twice AFAICT, so not a big deal, but wanted to mention it since it's a common pattern.